manifests/single_pod.go

```diff
@@ -160,30 +160,9 @@ func MainPodForIDM(m *v1alpha1.IDM, baseDomain string, defaultStorage string) *c
 						"--no-ntp",
 						"--no-sshd",
 						"--no-ssh",
-						"--verbose",
```

Comment:
Add this conditionally, if the debug indicator was provided when starting the controller.